lib/verify.js

```diff
@@ -208,6 +208,7 @@ function rebuildIndex (cache, opts) {
     }
     const buckets = {}
     for (const k in entries) {
+      /* istanbul ignore else */
       if (hasOwnProperty(entries, k)) {
```

Comment:
Another option would've been `for (const k of Object.keys(entries))`, to iterate over own props only.  Ignoring the else is fine, too, but I try to avoid having a loop where the entire body is an `if` block if I can help it.